src/i18n/Messages/Symfony/FlushInvalidatedResource.php

```diff
@@ -58,6 +59,7 @@ public function __unserialize(array $data): void
      */
     public function serialize()
     {
+        Deprecation::notice('4.12.0', 'Will be removed in 5.0');
```

Comment:
Add version to the phpdoc and update the message in both to be standardised

Reply:
Updated